docs/baseline-protocol-code/branch-naming.md

```diff
@@ -0,0 +1,23 @@
+# Branch Naming Standard
+
+## Main
+
+*main* - main development branch, feature and release branches branched from it, changes only through the PR process.
+
+## Feature
+
+*feature/XYZ-this-is-a-new-feature-branch* - in case related issue XYZ exists
+
+*feature/BLIP-#-repo-branch-standards* - in case related BLIP-# exists
+
+*feature/this-is-a-new-feature-branch* - in case no related item exists
+
+## Bugs
+
+*bugfix/name-of-the-bug*
```

Comment:
same question as above, also we should add the case when there is a issue opened

Reply:
Added